### ISSUE TITLE
Carousel - selected tab has no aria-selected="true"

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#14026]
Issue reported by user testing.

Environment Information
Win10
Chrome Version 83.0.4103.116
NVDA 2020.1

Description: The selected item in the tablist should be indicated by aria-selected=”true” for screen reader users. 

### expected behavior

Please see screenshot, every tab has aria-selected=false, but the selected tab should has aria-selected=true 

![image](https://user-images.githubusercontent.com/22620150/87446028-10f14f80-c5f9-11ea-8b5a-0b35c8508313.png)

**Comment by assuncaocharles** [comment]
Fixed by #14970
